test/binary-elementwise-operator-tester.h

```diff
@@ -179,6 +179,28 @@ class BinaryElementwiseOperatorTester {
     }
   }
 
+  int32_t Compute(int32_t a, int32_t b) const{
+    switch (operation_type()) {
+      case OperationType::Add:
+        return a + b;
+      case OperationType::CopySign:
+        return std::copysign(a, b);
+      case OperationType::Divide:
+        return a / b;
+      case OperationType::Maximum:
+        return std::max<int32_t>(a, b);
+      case OperationType::Minimum:
+        return std::min<int32_t>(a, b);
+      case OperationType::Multiply:
+        return a * b;
+      case OperationType::Subtract:
+        return a - b;
+      case OperationType::SquaredDifference:
+        return (a - b) * (a - b);
+      default:
+        return std::nan("");
```

Comment:
assert instead of returning nan
XNN_UNREACHABLE

Reply:
Added return statement to return MAX of value instead of nan. For XNN_UNREACHABLE need to include header files for the same. Is that fine?

Reply:
Yes that's fine